tests/h/schemas/base_test.py

```diff
@@ -119,3 +120,72 @@ def test_deserialize_raises_if_value_unknown(self, color_type):
     @pytest.fixture
     def color_type(self):
         return enum_type(Color)()
+
+
+class QueryParamSchema(colander.Schema):
+
+    int_field = colander.SchemaNode(colander.Integer(), missing=0)
+
+    string_field = colander.SchemaNode(colander.String(), missing=None)
+
+    list_field = colander.SchemaNode(colander.Sequence(),
+                                     colander.SchemaNode(colander.String()),
+                                     missing=None)
+
+    enum_field = colander.SchemaNode(colander.String(),
```

Comment:
I don’t see a test for enum?

Reply:
I'm not testing the validation behaviour exhaustively because that's covered by Colander's own tests. The tests here are focused on the custom logic added on top of Colander.